clusterloader2/testing/density/config.yaml

```diff
@@ -53,14 +53,21 @@ steps:
       nodeMode: {{$NODE_MODE}}
       resourceConstraints: {{$DENSITY_RESOURCE_CONSTRAINTS_FILE}}
 # Create saturation pods
+- measurements:
+  - Identifier: SaturationPodStartupLatency
+    Method: PodStartupLatency
+    Params:
+      action: start
+      labelSelector: group = saturation
+      threshold: {{$saturationRCTimeout}}s
 - measurements:
   - Identifier: WaitForRunningSaturationRCs
     Method: WaitForControlledPodsRunning
     Params:
       action: start
       kind: ReplicationController
       labelSelector: group = saturation
-      operationTimeout: {{$saturationRCTimeout}}s
+      operationTimeout: 15m
```

Comment:
This will be too low in large clusters (where we create all RCs at the very beginning, but creating all pods take 1h+)